docs/customizations/custom-css.md

```diff
@@ -22,6 +22,43 @@ After entering your rules, click on the "Submit" button to save your changes. Yo
 
 ![homarr custom css result](./img/customizations-custom-css-result.png)
 
+### Example: Background image blur
+
+``` 
+body {
+  backdrop-filter: blur(60px);
+  -webkit-backdrop-filter: blur(60px);
+}
+```
+Adding the above to your custom CSS will give you blur support for both desktop and mobile. You can change how much blur is wanted by changing the (60px) to a choice that better suits your needs. 
+
+![homarr custom css results](./img/customizations-custom-css-blur.png)
+
+We can also add blur support for the settings modal. Here is an example of how we achieve this.
+
+```
+.mantine-w4dwz9 {
```

Comment:
Can you mention, that these class names will change each time Homarr is updated?
Also, you could create a ticket for this, so we crate unused static class names, so they don't change anymore.